server/config/configresource/config_resource.go

```diff
@@ -54,245 +60,102 @@ func Repository(db *sql.DB, opts ...option) *repository {
 
 }
 
-const ResourceID = "/app/config/update"
+const (
+	ResourceID   = "/app/config/update"
+	ResourceName = "Config"
+)
 
 type publisher interface {
 	Publish(resourceID string, message any)
 }
 
-type repository struct {
+type Repository struct {
 	db        *sql.DB
 	publisher publisher
 }
 
-func (r *repository) publish(config Config) {
+func (r *Repository) publish(config Config) {
 	if r.publisher == nil {
 		return
 	}
 
 	r.publisher.Publish(ResourceID, config)
 }
 
-func (r *repository) Current(ctx context.Context) Config {
+func (r *Repository) Current(ctx context.Context) Config {
 	defaultConfig := Config{
 		AnalyticsEnabled: true,
 	}
 
-	list, err := r.List(ctx, 1, 0, "", "", "")
-	if err != nil || len(list) != 1 {
+	cfg, err := r.Get(ctx, id.ID("current"))
```

Comment:
the id doesn't really exists, so I just picked `current` so we can have a non empty id that makes sense, for consistency